src/DataStructures/DataBox/Prefixes.hpp

```diff
@@ -47,4 +47,45 @@ struct dt : db::DataBoxPrefix {
   using type = db::item_type<Tag>;
   using tag = Tag;
 };
+
+/// \ingroup DataBoxTags
+/// \brief Prefix indicating a flux
+template <typename Tag, typename VolumeDim, typename Fr,
+          typename = std::nullptr_t>
```

Comment:
Is there any situation in which this tag can be used where db::item_type<Tag> is not a Tensor?  If there are none, why not add a Requires?

Reply:
Not that I can think of. I'll add a requires.